app/app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <!-- Default screen margins, per the Android Design guidelines. -->
+    <dimen name="activity_horizontal_margin">16dp</dimen>
```

Comment:
Pls try to use a utils class and avoid hardcode

Reply:
hi, it's not hardcode, it's a dimension configuration on a dimen.xml file

Reply:
it's okey, my mistake sorry